cpp/src/slice2cs/Gen.cpp

```diff
@@ -83,28 +83,35 @@ namespace
         return "???";
     }
 
-    string getDeprecateReason(const ContainedPtr& p1, const ContainedPtr& p2, const string& type)
+    string getDeprecateReason(const ContainedPtr& p1, const string& type)
```

Comment:
I don't know if we should generate a special doc-comment message for deprecated operation and the like.

But we should at least give this helper function a clearer/consistent name, like `getDocCommentDeprecationReason`.